Module 2

life-invader opened the pull request.

life-invader committed:
3.3 Добавляет компонент поиска

life-invader committed:
3.4 Добавляет стор
- И загрузку данных

life-invader committed:
3.5  Добавляет несколько ключевых изменений
- Лоадер на время загрузки
- Компонент ошибки на случай неудачи загрузки
- Добавляет инстанс axios

life-invader committed:
3.6 Приводит код в порядоку
- Удаляет комментарии и прочее

tootsieroll commented on src/components/catalog/catalog.tsx:
```diff
@@ -1,23 +1,65 @@
+              type="button"
+              onClick={showMoreClickHandler}
+            >
+              Показать еще 5 билетов
```
тут можно добавить отображение реального количества оставшихся билетов если их меньше 5

tootsieroll commented:
Оставила коммент ,что можно улучшить, а в остальном проект сделан хорошо, других замечаний нет

life-invader committed:
3.7 Исправлчет замечания + Readme
- Обновляет readme
- Атрибуты for для label
- Обновление компонента кнопки "показать больше"

life-invader committed:
3.8 Добавляет описание типов компонента
- Для компонента кнопки 'показать еще'

life-invader committed:
3.9 Обновляет компоненты 'Лоадер' и 'Ошибка'

life-invader committed:
Merge branch 'module-1' into module-2
